### PR TITLE
Restructuring the page

### PR DESCRIPTION
**Pull request by somindatommy**
## Purpose
Restructuring the page [1].

[1] - https://is.docs.wso2.com/en/5.9.0/setup/setting-up-separate-databases-for-clustering/

### CODE REVIEW

File: en/docs/setup/setting-up-separate-databases-for-clustering.md [hunk 1]
```diff
@@ -4,125 +4,91 @@ This section guides you through the logical separation of data that you
...
-use one central database for config and governance registry mounts. By
+user management details and identity data. By
 default, each WSO2 product is shipped with an embedded H2 database that
 works for all types of data.
 
-!!! warning "Embedded H2 is not recommended in production"
-    
-    The embedded H2 database is NOT recommended in enterprise testing and
+!!! warning "Embedded H2 is NOT RECOMMENDED in production"
+    The embedded H2 database is **NOT RECOMMENDED** for enterprise testing and for
```
Comment (gomathyk) [inline en/docs/setup/setting-up-separate-databases-for-clustering.md]: Please remove for

File: en/docs/setup/setting-up-separate-databases-for-clustering.md [hunk 2]
```diff
@@ -4,125 +4,91 @@ This section guides you through the logical separation of data that you
...
-    topic in the product administration guide.
-    
+    !!! note
+        You can use the embedded H2 database in development environments. For more information on
+        databases, see [Working with Databases](../../administer/working-with-databases/).
 
-You can create the following databases and associated datasources. This
-is **NOT** mandatory and you can choose to not create these databases if
-you wish and simply have a single database to handle all these concerns.
+You can create the following databases logical separation of data. This
```
Comment (gomathyk) [inline en/docs/setup/setting-up-separate-databases-for-clustering.md]: for the logical separation of data

File: en/docs/setup/setting-up-separate-databases-for-clustering.md [hunk 3]
```diff
@@ -4,125 +4,91 @@ This section guides you through the logical separation of data that you
...
+is **NOT mandatory** and you can choose not to create these databases if
+you wish have default databases to handle all these concerns.
```
Comment (gomathyk) [inline en/docs/setup/setting-up-separate-databases-for-clustering.md]: wish to have

File: en/docs/setup/setting-up-separate-databases-for-clustering.md [hunk 4]
```diff
@@ -4,125 +4,91 @@ This section guides you through the logical separation of data that you
...
-<td><code>             &lt;IS_HOME&gt;/repository/conf/datasources/master-datasources.xml            </code></td>
-</tr>
-<tr class="odd">
-<td><code>             METRICS_DB            </code></td>
-<td>Metrics Database is used to store the runtime metrics data reported by the Metrics JDBC Reporter periodically. There are 5 tables to store metrics data for each metric type. The metric types are Counter, Meter, Gauge, Histogram, and Timer.</td>
-<td><pre><code>&lt;IS_HOME&gt;/dbscripts/</code></pre></td>
-<td><code>             &lt;IS_HOME&gt;/repository/conf/datasources/metrics-datasources.xml            </code></td>
+<td>
+<li>Contains identity related data. For example, OAuth 2.0, SAML 2.0, User Managed Access (UMA) etc.</li>
+<li>In the default database configuration, all the UMA, concent and identity data are located <code>IDENTITY_DB</code>.
```
Comment (gomathyk) [inline en/docs/setup/setting-up-separate-databases-for-clustering.md]: are located in the

File: en/docs/setup/setting-up-separate-databases-for-clustering.md [hunk 5]
```diff
@@ -4,125 +4,91 @@ This section guides you through the logical separation of data that you
...
+ In a deplyment both identity and UMA can be seperated for a single db while concent data is configured to a seperate
```
Comment (gomathyk) [inline en/docs/setup/setting-up-separate-databases-for-clustering.md]: consent

File: en/docs/setup/setting-up-separate-databases-for-clustering.md [hunk 6]
```diff
@@ -4,125 +4,91 @@ This section guides you through the logical separation of data that you
...
 <tr class="odd">
 <td><code>             REGISTRY_DB            </code></td>
-<td>Shared database for config and governance registry mounts in the product's nodes. This includes data on tenants and keystores.</td>
-<td><pre><code>&lt;IS_HOME&gt;/dbscripts/</code></pre></td>
-<td><code>             &lt;IS_HOME&gt;/repository/conf/datasources/master-datasources.xml            </code></td>
+<td>
+<li>Shared database for config and governance registry mounts in the product's nodes. This includes data on tenants and 
+keystores.</li>
+<li>In the default database configuration, user data and registry data is located inside <code>SHARED_DB</code>. 
+During a deployment user data can be configured to a different db while registry data remain in the <code>SHARED_DB</code>
```
Comment (gomathyk) [inline en/docs/setup/setting-up-separate-databases-for-clustering.md]: During a deployment, 
while registry data remains

File: en/docs/setup/setting-up-separate-databases-for-clustering.md [hunk 7]
```diff
@@ -4,125 +4,91 @@ This section guides you through the logical separation of data that you
...
-#### Configuring the datasources
-
-Configure the datasources for the databases that you created above in
-both the WSO2 Identity Server nodes of your cluster. For instructions on
-how to configure the datasources for the databases you created, see
-[Changing the Carbon
-Database](../../administer/changing-the-carbon-database).
+!!! note
+    For more information on `Shared db` and `Identity db`, 
+    see [Working with Databases](#../../administer/working-with-databases/).
```
Comment (gomathyk) [inline en/docs/setup/setting-up-separate-databases-for-clustering.md]: remove hash in the relative path

Comment (somindatommy) [reply en/docs/setup/setting-up-separate-databases-for-clustering.md]: Fixed: 55e7088941e0c75d2805c48dc949bcff9db0324c

File: en/docs/setup/setting-up-separate-databases-for-clustering.md (hunk 6 above)
Comment (somindatommy) [reply en/docs/setup/setting-up-separate-databases-for-clustering.md]: Fixed: 55e7088941e0c75d2805c48dc949bcff9db0324c

File: en/docs/setup/setting-up-separate-databases-for-clustering.md (hunk 5 above)
Comment (somindatommy) [reply en/docs/setup/setting-up-separate-databases-for-clustering.md]: Fixed: 55e7088941e0c75d2805c48dc949bcff9db0324c

File: en/docs/setup/setting-up-separate-databases-for-clustering.md (hunk 4 above)
Comment (somindatommy) [reply en/docs/setup/setting-up-separate-databases-for-clustering.md]: Fixed: 55e7088941e0c75d2805c48dc949bcff9db0324c

File: en/docs/setup/setting-up-separate-databases-for-clustering.md (hunk 3 above)
Comment (somindatommy) [reply en/docs/setup/setting-up-separate-databases-for-clustering.md]: Removed

File: en/docs/setup/setting-up-separate-databases-for-clustering.md (hunk 2 above)
Comment (somindatommy) [reply en/docs/setup/setting-up-separate-databases-for-clustering.md]: Removed

File: en/docs/setup/setting-up-separate-databases-for-clustering.md (hunk 1 above)
Comment (somindatommy) [reply en/docs/setup/setting-up-separate-databases-for-clustering.md]: Fixed: 55e7088941e0c75d2805c48dc949bcff9db0324c

**Pull request merged**